server/src/test/java/org/elasticsearch/search/slice/SliceBuilderTests.java

```diff
@@ -461,7 +462,7 @@ public void testSerializationBackcompat() throws IOException {
 
         SliceBuilder copy62 = copyWriteable(sliceBuilder,
                 new NamedWriteableRegistry(Collections.emptyList()),
-                SliceBuilder::new, Version.V_6_2_0);
+                SliceBuilder::new, VersionUtils.getPreviousVersion(Version.V_6_3_0));
```

Comment:
Isn't this just removed in your 6.3 removal PR?

Reply:
let it be that PR's worry :)